client/src/user/dashboard/Career.jsx

```diff
@@ -6,6 +6,7 @@ import Chip from "@mui/material/Chip";
 import { FaLinkedin, FaXTwitter, FaInstagram } from "react-icons/fa6";
 import { ToastContainer, toast } from "react-toastify";
 import { submitUserFormData } from "../../../api";
+import TagFacesIcon from '@mui/icons-material/TagFaces';
```

Comment:
Why is this added?

Reply:
It is used below in code but was not added!